docs/installation.rst

```diff
@@ -144,69 +126,36 @@ hardware intended to run openfe jobs, as it will test GPU specific features.
 
 With that, you should be ready to use ``openfe``!
 
-``conda-lock`` file
--------------------
-
-.. _conda-lock: https://github.com/conda/conda-lock?tab=readme-ov-file#conda-lock
-
-A `conda-lock`_ file is a cross platform way of specifying a conda environment that specifies packages in a reproducible way.
-Unlike the single file installer, an internet connection is required to install from a ``conda-lock`` file.
-We recomend the use of a ``conda-lock`` file when the same conda environment is required across different systems.
-
-
-.. note::
-
-   You will likely need to install ``conda-lock``.
-   We strongly recomend installing ``conda-lock`` in a new virtual environment.
-   This will reduce the chance of dependency conflicts ::
-
-       $ # Install conda lock into a virtual environment
-       $ conda create -n conda-lock -c conda-lock
-       $ # Activate the environment to use the conda-lock command
-       $ conda activate conda-lock
-
-See https://github.com/conda/conda-lock?tab=readme-ov-file#conda-lock for more information on ``conda-lock``.
-
-The latest version of the `conda-lock` file we provide can be downloaded with ::
-
-  $ curl -LOJ https://github.com/OpenFreeEnergy/openfe/releases/latest/download/openfe-conda-lock.yml 
+Installation with ``mamba``
+---------------------------
 
-If a particular version is required, the URL will look like this (using the ``openfe 1.0.1`` release as an example) ::
+If you already have a `Mamba <https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html>`_
+(or `Micromamba <https://mamba.readthedocs.io/en/latest/installation/micromamba-installation.html>`_ ) installation, you can install ``openfe`` with:
 
-  $ curl -LOJ https://github.com/OpenFreeEnergy/openfe/releases/download/v1.0.1/openfe-1.0.1-conda-lock.yml
+.. parsed-literal::
 
-Create a conda environment from the lock file and activate it::
+  mamba create -c conda-forge -n openfe_env openfe=\ |version|
+  mamba activate openfe_env
 
-  $ conda-lock install -n openfe openfe-conda-lock.yml
-  $ conda activate openfe
+Note that you must run the latter line in each shell session where you want to use ``openfe``. OpenFE recommends the Mamba package manager for most users as it is orders of magnitude faster than the default Conda package manager. Mamba is a drop in replacement for Conda.
 
 .. note::
+   If you plan on using GAFF to parametrize your system, you **must** install ``openmmforcefields`` version ``0.12.0``.
```

Comment:
This is incorrect - I believe openmmforcefields 0.14.1 brought back gaff support fully.

Reply:
Yes this hasn't been true since we released 0.14